### ISSUE TITLE
Setup `parse_place` and add formatting output for it (break into country and landmark)

### ISSUE THREAD

**Issue by gannonprudhomme** [gannonprudhomme/CSCE-482-KNOW-backend#52]
- Add `parse_landmark` and `parse_country` to `AbstractParser` (base outlines, not an actual implementation)
- Add `parse_landmark` and `parse_country` to `WikidataParser` (the actual implementation)
  - write tests for the whole thing (e.g. integration) that actually takes some real URI, does the request to the endpoint, then formats it
- Add a `format_landmark` and `format_country` functions and add tests for them
  - Note that it should use the `format_output` function from `rdf/parser/format_output`
  - As in, just take the expected output from the query endpoint then assert that it formats it right
  - Basically, don't make an actual API call when you test it
